src/com/spacegame/game/GameState.java

```diff
@@ -0,0 +1,50 @@
+package com.spacegame.game;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Created by jparrott on 9/11/2016.
+ */
+public class GameState {
+
+    private final static Logger log = LogManager.getLogger(GameState.class);
+
+    private GameState(){
+
+    }
+
+    private static GameState instance = null;
+
+    public static GameState getInstance() {
```

Comment:
Just a note, this method (i.e., `GameState.getInstance()`)   should be used through out the game to get the current GameState object